src/algebra/category/Group/adjunctions.lean

```diff
@@ -57,3 +72,23 @@ example {G H : AddCommGroup.{u}} (f : G ⟶ H) [mono f] : function.injective f :
 (mono_iff_injective f).1 (right_adjoint_preserves_mono adj (by apply_instance : mono f))
 
 end AddCommGroup
+
+namespace abelianization
+
+/-- The abelianization functor `Group ⥤ CommGroup` sending a group `G` to its abelianization `Gᵃᵇ`.
+ -/
+def abelianize : Group.{u} ⥤ CommGroup.{u} :=
```

Comment:
This will now have name `abelianization.abelianize`...
How about just calling this `abelianize` and the adjunction `abelianize_adj`?

Reply:
Thanks. Done.